spec/api-crash-reporter-spec.js

```diff
@@ -50,6 +50,7 @@ describe('crashReporter module', function () {
         assert.equal(fields['platform'], process.platform)
         assert.equal(fields['extra1'], 'extra1')
         assert.equal(fields['extra2'], 'extra2')
+        assert.equal(fields['extra3'], 'extra3')
```

Comment:
Instead of adding this assertion here, it would seem more comprehensive to add a second spec that calls `setExtraParameters`  explicitly with different values for `extra1` and `extra2` to verify that `setExtraParameters overrides the `start` extra params.

Then we'll be sure the extra parameters work in both cases, specified to `start` and set explicitly via `setExtraParameters`.